### DISCUSSION TITLE
Archive of XML VOEvents

### DISCUSSION THREAD

**Discussion by aleberti** [chime-frb-open-data/community#53] [General]
Dear all,

I have a simple question: is there an archive of the VOEvents sent in XML format somewhere? For example, for the VOEvents sent by the GCN, I use http://voeventdb.4pisky.org/apiv1/ (it has also a Python interface if needed), so that I can check past events, if needed. So, is there something similar available for FRB VOEvents (directly from the CHIME collaboration, or also from external sources), or any future plan?

Thank you and best regards,
Alessio Berti

**Comment by andrewzwaniga** [comment]
Hi Alessio,

Thanks for your question. Currently CHIME/FRB does not host a publicly-accessible archive of previously published FRB VOEvents in XML format. While we do plan to establish something akin to this in the future, during the current phase of operation we are providing the XMLs in real-time only. I appreciate that this would be useful for many in the community and will keep this in mind as we prioritize improvements to the VOEvent Service administration and delivery. 

Best regards,
Andrew

**Comment by corbettht** [comment]
Hi Andrew,

Any chance of an update on this? Or really, any kind of access to previously-released public events - a CSV file or similar listing would be great. Historical data would be a huge help to folks looking through archival events for coincidence searches. Personally, even a really noisy (many FP) listing would be fine. 

Thanks again for this repo - it's an amazing resource!

Cheers,
hank
